assets/src/dashboard/app/views/editorSettings/index.js

```diff
@@ -407,29 +402,16 @@ function EditorSettings() {
         )}
         title={__('Are you sure you want to remove this logo?', 'web-stories')}
         onClose={() => setActiveDialog(null)}
-        actions={
-          <>
-            <Button
-              type={BUTTON_TYPES.TERTIARY}
-              onClick={() => setActiveDialog(null)}
-              size={BUTTON_SIZES.SMALL}
-            >
-              {__('Cancel', 'web-stories')}
-            </Button>
-            <Button
-              type={BUTTON_TYPES.PRIMARY}
-              onClick={handleDialogConfirmRemoveLogo}
-              size={BUTTON_SIZES.SMALL}
-            >
-              {__('Delete Logo', 'web-stories')}
-            </Button>
-          </>
-        }
+        closeText={__('Cancel', 'web-stories')}
+        onConfirm={handleDialogConfirmRemoveLogo}
+        confirmText={__('Delete Logo', 'web-stories')}
       >
-        {__(
-          'The logo will be removed from any stories that currently use it as their publisher logo.',
-          'web-stories'
-        )}
+        <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL} as="p">
```

Comment:
Isn't the default a `p` tag?
```suggestion
        <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}>
```

Reply:
thanks, yeah - I'd originally added these for readability but those defaults are there for a reason, you're right. I'll remove 'em!